config/docker-compose.yml

```diff
@@ -1,19 +1,26 @@
 version: '3.5'
 
+volumes:
+  postgres-data:
+  elasticsearch-data:
```

Comment:
What is dmrunner doing with its data currently?

Reply:
Currently it's stored in the container, so when you change the image you lose the data

Reply:
So if we actually did want to wipe the data, how would we go about it? Maybe worth sticking something in the README?

Reply:
I added something to the commit so the `make data` command will wipe as before